### PR TITLE
 Additional explanation on setting up network with CA

### PR DESCRIPTION
**Pull request by mcr222**
Signed-off-by: Marc Cayuela Rafols

### CODE REVIEW

**Comment by mcr222** [comment]
Signed-off-by: Marc Cayuela Rafols

File: docs/dev-setup/devnet-setup.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ By default, we are using a consensus plugin called `NOOPS`, which doesn't really
 docker run --rm -it -e OPENCHAIN_VM_ENDPOINT=http://172.17.0.1:4243 -e OPENCHAIN_PEER_ID=vp0 -e OPENCHAIN_PEER_ADDRESSAUTODETECT=true openchain-peer obc-peer peer
 ```
 
+If started with security, enviroment variables regarding security enabling, CA address and peer's ID and password have to be changed:
+
+```
+docker run --rm -it -e OPENCHAIN_VM_ENDPOINT=http://172.17.0.1:4243 -e OPENCHAIN_PEER_ID=vp0 -e OPENCHAIN_PEER_ADDRESSAUTODETECT=true -e OPENCHAIN_SECURITY_ENABLED=true -e OPENCHAIN_SECURITY_PRIVACY=true -e OPENCHAIN_PKI_ECA_PADDR=172.17.0.1:50051 -e OPENCHAIN_PKI_TCA_PADDR=172.17.0.1:50051 -e OPENCHAIN_PKI_TLSCA_PADDR=172.17.0.1:50051 -e OPENCHAIN_SECURITY_ENROLLID=vp0 -e OPENCHAIN_SECURITY_ENROLLSECRET=XX  openchain-peer obc-peer peer
```
Comment (genggjh) [inline docs/dev-setup/devnet-setup.md:56]: I think we need change the OPENCHAIN to CORE.

File: docs/dev-setup/devnet-setup.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ See [Logging Control](logging-control.md) for information on controlling
 logging output from the `obc-peer` and chaincodes.
 
 
-**Note:** When running with security enabled, follow the security setup instructions described in [Chaincode Development](https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional) to set up the CA server and log in registered users before sending chaincode transactions.
+**Note:** When running with security enabled, follow the security setup instructions described in [Chaincode Development](https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional) to set up the CA server and log in registered users before sending chaincode transactions.  In this case peers started using Docker images need to point to the correct CA address (default is localhost). CA addresses have to be specified in openchain.yaml variables paddr of eca, tca and tlsca.
```
Comment (genggjh) [inline docs/dev-setup/devnet-setup.md:10]: Should change file path:
https://github.com/openblockchain/obc-docs/blob/master/api/SandboxSetup.md#security-setup-optional
->
https://github.com/hyperledger/fabric/blob/master/docs/API/SandboxSetup.md#security-setup-optional

And change the file name:
openchain.yaml
->
core.yaml

**Comment by mcr222** [comment]
Oh! Then a lot of the documentation should be updated because OPENCHAIN references are everywhere (as well as links to documentation in openblockchain). I guess I will wait until the merge is done completely.

**Comment by genggjh** [comment]
I think so :-).

A lot of changes recently.

**Comment by angrbrd** [comment]
Thanks @mcr222 and @genggjh. We are looking to have the merge done today and will update as much of the docs as we can spot :-). Thanks for your patience.

**Comment by binhn** [comment]
@mcr222 I am not sure what is going on in this pull request. We were in the middle of moving the code to this location.  Would you mind close and recreate as appropriate?

**Pull request closed without merging**